recipe/meta.yaml

```
@@ -35,150 +33,72 @@ requirements:
-        - autoconf
-        - automake
-        - make
-      host:
-        - zlib
-        - llvmdev {{ llvm_version }}.*
-        - llvm {{ llvm_version }}.*
-        - tapi <1100   # [not (arm64 or linux)]
-        - tapi >=1100  # [arm64 or linux]
-        - libcxx  # [osx]
-        #- {{ pin_subpackage("ld64_" + cross_platform, max_pin="x.x") }}
-      run:
-        - libcxx  # [osx]
-        - {{ pin_subpackage("ld64_" + cross_platform, max_pin="x.x") }}
-        - ldid    # [osx]
-      run_constrained:
-        #- ld64 {{ ld64_version }}.*
-        - cctools {{ cctools_version }}.*
-    test:
-      commands:
-        # For arm64, cctools as calls the clang integrated assembler. Don't check for it
-        - test -f $PREFIX/libexec/as/x86_64/as
-        - test -f $PREFIX/libexec/as/arm/as        # [arm64]
-        - test -f $PREFIX/bin/{{ macos_machine }}-as
-        - test -f $PREFIX/bin/{{ macos_machine }}-ranlib
-        - test -f $PREFIX/bin/{{ macos_machine }}-ld    # [not linux]
-        - test -f $PREFIX/bin/{{ macos_machine }}-ar
-        - test -f $PREFIX/bin/{{ macos_machine }}-otool
-        # Check that otool is functioning
-        - $PREFIX/bin/{{ macos_machine }}-otool -l $PREFIX/bin/{{ macos_machine }}-otool  # [target_platform == build_platform and target_platform == cross_platform]
-        - test -f $PREFIX/bin/{{ macos_machine }}-install_name_tool
-        - test -f $PREFIX/bin/{{ macos_machine }}-strip
-    about:
-      home: https://github.com/tpoechtrager/cctools-port
-      license: APSL-2.0
-      license_family: Other
-      license_file: cctools/APPLE_LICENSE
-      summary: Assembler, archiver, ranlib, libtool, otool et al for Darwin Mach-O files
-
-  - name: ld64_{{ cross_platform }}
+  - name: ld64
     version: {{ ld64_version }}
-    script: install-ld64.sh
     build:
-      activate_in_script: True
+      skip: True   # [cross_platform != target_platform]
       missing_dso_whitelist:
-        - /usr/lib/libxar.1.dylib  # [osx]
+        - "*/libc++.1.dylib"  # [osx]
+        - "*/libtapi.dylib"  # [osx]
+    script: install-ld64.sh
     requirements:
-      build:
-        - {{ compiler('cxx') }}
-        - autoconf
-        - automake
-        - make
       host:
-        - llvm  {{ llvm_version }}.*
-        - clang {{ llvm_version }}.*
-        - tapi <1100   # [not (arm64 or linux)]
-        - tapi >=1100  # [arm64 or linux]
-        - libcxx  # [osx]
-        - libuuid  # [linux]
+        - llvm  {{ llvm_version }}
+        - clang {{ llvm_version }}
+        - libcxx-devel {{ llvm_version }}
+        - tapi 1100.0.11
       run:
-        - {{ pin_compatible("tapi") }}
-        - libcxx  # [osx]
-        - ldid    # [osx]
+        - tapi
+        - libcxx-devel {{ llvm_version }}
+        - __osx >={{ MACOSX_DEPLOYMENT_TARGET }}  # [osx]
       run_constrained:
-        - {{ pin_compatible("clang") }}
-        - ld {{ ld64_version }}.*
         - cctools {{ cctools_version }}.*
```

```suggestion
        - {{ pin_compatible("clang") }}
        - ld {{ ld64_version }}.*
        - cctools {{ cctools_version }}.*
```
I believe these are good to keep. ld in case it is available on another channel.